sunpy/net/hek/hek.py

```diff
@@ -88,18 +114,38 @@ def search(self, *query):
             ndata.append(new)
 
         if len(ndata) == 1:
-            return self._download(ndata[0])
+            return HEKResponse(self._download(ndata[0]), client=self)
         else:
-            return self._merge(self._download(data) for data in ndata)
+            return HEKResponse(self._merge(self._download(data) for data in ndata), client=self)
 
     def _merge(self, responses):
         """ Merge responses, removing duplicates. """
         return list(unique(chain.from_iterable(responses), _freeze))
 
+    def fetch(self, *args, **kwargs):
```

Comment:
Should have a docstring just for completeness just explaining it's a noop

Reply:
Fixed.